server/gql/models/users.js

```diff
@@ -0,0 +1,84 @@
+import { GraphQLID, GraphQLInt, GraphQLNonNull, GraphQLObjectType, GraphQLString } from 'graphql';
+import { getNode } from '@gql/node';
+import { createConnection } from 'graphql-sequelize';
+import { timestamps } from './timestamps';
+import db from '@database/models';
+import { totalConnectionFields } from '@utils/index';
+
+const { nodeInterface } = getNode();
+
+export const userFields = {
+  id: { type: GraphQLNonNull(GraphQLID) },
+  firstName: { type: GraphQLNonNull(GraphQLString) },
+  lastName: { type: GraphQLNonNull(GraphQLString) },
+  email: { type: GraphQLNonNull(GraphQLString) },
+  password: { type: GraphQLNonNull(GraphQLString) }
+};
+
+const User = new GraphQLObjectType({
+  name: 'user',
+  interfaces: [nodeInterface],
+  sqlPaginate: true,
```

Comment:
you don't need sqlPaginate anymore, we can remove this from everywhere. 

this is a remnant from the time we used join-monster